src/index.js

```diff
@@ -480,6 +490,89 @@ export default class AuthnWidget {
     document.querySelector('#submit').disabled = disabled;
   }
 
+  async registerIdVerificationRequiredEventHandler() {
+    let data = this.store.getStore();
+
+    if (data.errorDetails !== undefined)
+    {
+      document.getElementById("requiredDescription").style.display = "none";
+      document.getElementById("errorDescription").style.display = "block";
+
+      document.getElementById("errorBlock").style.display = "flex";
+      document.getElementById("errorMessage").innerHTML = this.makeIdVerificationErrorMessage(data.errorDetails);
```

Comment:
This looks pretty similar to how the errors.hbs is laid out. Do you know if the normal flow is not displaying them correctly and we need something custom here?

Reply:
I thought errors.hbs (which makes use of `userMessages`) can only be used to display error that came back from an action? Would it work if the errorDetails in the state response has this format?

```
"errorDetails": [
   {
     "code": "PINGONE_VERIFY_ERROR_520",
     "message": "Extraction acceptance failed",
     "userMessage": "Third party service check failed.",
     "providerId": "Mitek"
   }
 ]
```

Reply:
Hmm good question. I believe we changed the ability to display all the error details as errors with the registration case.

In store.js#getErrorDetails method, Looks like we limit that capability to specific error types?

Reply:
I think that's one if the issue (not one of the specified type of error). The other one would be that the format of the error that is being returned is a bit different as well.

For example if I were to login using incorrect password the error the widget receives looks like this
```
{
   "code":"VALIDATION_ERROR",
   "message":"“One or more validation errors occurred.“",
   "userMessage":"“One or more validation errors occurred.“",
   "details":[
      {
         "code":"CREDENTIAL_VALIDATION_FAILED",
         "message":"Credential validation failed.",
         "userMessage":"We didn't recognize the username or password you entered. Please try again."
      }
   ]
}
```
But the full response of a verify state with error looks like
```
{
   "id":"LGvI8",
   "pluginTypeId":"4Qx913wUI9wh_IGt0inNXg",
   "status":"ID_VERIFICATION_FAILED",
   "errorDetails":[
      {
         "code":"PINGONE_VERIFY_ERROR_LIMIT_EXCEEDED",
         "message":"",
         "userMessage":"Max ID verifications has been exceeded. Please try again later.",
         "providerId":"ID_VALIDATION"
      }
   ],
   "_links":{
      "…"
   }
}
```
Instead of checking `json.code` it needs to check for `json.status` and instead of reading `json.details` the widget would have to check `json.errorDetails`.

It won't be hard to change the code in `store.js`, just wondering if you think if `getErrorDetails` should be changed to accommodate errors returned like this? Or should it be left as something custom in `index.js`?

Reply:
Oh I see, we are passing the API response from the verify service through?

I'd prefer that we make errors hbs able to handle this scenario. It seems like this might become more and more common in the future.

Reply:
I couldn't get the errors to display properly using what you suggested exactly, the newest commit it what I have attempted that uses `{{> errors}}`. Let me know what you think 🙇

Reply:
Yeah, I think that's as good as it gets. Thanks for making those changes